build.sbt

```diff
@@ -668,20 +668,13 @@ lazy val commonSettings = Def.settings(
   })
 )
 
-lazy val enforceMimaCompatibility = true // Enable / disable failing CI on binary incompatibilities
+lazy val enforceMimaCompatibility = false // Enable / disable failing CI on binary incompatibilities
```

Comment:
PS: I disabled this since I had to add too many filters due to the removed methods